### ISSUE TITLE
meta: support for podman

### ISSUE THREAD

**Issue by seanfarley** [riscy/melpazoid#20]
This is probably a huge ask but would it be possible to support [podman](https://podman.io)? Trying to run this locally gives:

```
python3 melpazoid/melpazoid.py
Using local repository at /Users/sean/projects/org-cv
<!-- Building container for ox-altacv... 🐳 -->


Stderr output while compiling/loading:
\ ```
make[1]: docker: No such file or directory
make[1]: *** [image] Error 1
\ ```

Package and license:
- Prefer the default recipe or `:defaults`, if possible.
- GNU General Public License v3.0 (via API)
- GPL* -- org-cv-utils.el: Common utility functions for CV exporters
- GPL* -- ox-altacv.el: LaTeX altacv Back-End for Org Export Engine
```

I'm totally willing to try to help but wanted to bring up this discussion first!

**Comment by riscy** [comment]
Actually this is something I've been wanting to try! It became a lot easier when [podman desktop](https://podman-desktop.io/) was released.

**Comment by seanfarley** [comment]
Is this something that others like me can help with or is it more of something you have a handle on?

**Comment by riscy** [comment]
I think I've got a handle on it - I successfully tested out podman desktop last week, and with the holidays I've got some more free time.  Ultimately I have to make sure that melpazoid can continue to run on my aging laptop.

**Comment by seanfarley** [comment]
Just checking back to see if anymore progress was made :-) No worries if not, I've been really, really busy myself!

**Comment by riscy** [comment]
Apologies for the long wait for a reply.   Obviously I never got around to this during the holidays. :)

I'm planning on taking another look today, but if you end up developing something at the same time and it's a good fit, I'd happily merge it here.

**Comment by seanfarley** [comment]
Oh, no worries at all :-) I've been distracted as well!

**Comment by riscy** [comment]
I believe everything needed for podman was merged with #27 -- but let me know if not; and thank you for bringing this up. :)

**Comment by seanfarley** [comment]
Yay! Thanks for doing this!
